Oscar Awards/Logic.java

```diff
@@ -0,0 +1,70 @@
+import java.io.*;
+public class Logic {
+    public static void main(String args[])
+    {
+        try  
+        {  
+            // String path="/home/nikhildewoolkar/Desktop/RedHat_Work/CSV_TO_JSON/the_oscar_award.csv";
+            File file=new File("the_oscar_award.csv");//path
```

Comment:
I would suggest using Java 11+ IO APIs. for example, you can read all lines of a file using `Files.lines(Paths.get("path to file")`

Reply:
Done